packages/gatsby-transformer-json/src/__tests__/gatsby-node.js

```diff
@@ -26,18 +26,18 @@ describe(`Process JSON nodes correctly`, () => {
     node.content = JSON.stringify(data)
 
     const createNode = jest.fn()
-    const updateNode = jest.fn()
-    const boundActionCreators = { createNode, updateNode }
+    const addChildNodeToParentNode = jest.fn()
```

Comment:
This one already existed, but I can't help to suggest making this one shorter too 🙂  If you add a node to a parent it is obviously a child. So `addChildNode` or `addNodeToParent` would have been enough imo.